.circleci/config.yml

```diff
@@ -84,6 +84,31 @@ references:
       path: test-reports
       destination: test-reports
 
+  benchmark: &benchmark
+    run:
+      name: Run benchmarks
+      command: |
+          pip install --user gitpython
+          echo 'export PYTHONPATH=$PYTHONPATH:.' >> $BASH_ENV
+          source $BASH_ENV
```

Comment:
Is there a reason why you don't just call `export PYTHONPATH=$PYTHONPATH:.` directly?

Reply:
https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command seems to recommend doing so.